integrations/discord/websockets/save.go

```diff
@@ -1,12 +1,15 @@
-package discord
+package websockets
```

Comment:
Nit: given that Discord supports only WS events, unlike Slack that has 2 modes, I think it makes sense to move the files in this package to to `discord` and remove the `websockets` package

Reply:
Done